src/Auth.php

```diff
@@ -159,7 +158,12 @@ public function getAuthHeaders() {
             $headers[] = 'Authorization: Basic ' . base64_encode( $this->defaultAuthOptions->key );
         } else {
             $this->authorizeInternal( [], [], $force = false ); // authorize only if necessary
-            $headers[] = 'Authorization: Bearer '. base64_encode( $this->tokenDetails->token );
+            $headers[] = 'Authorization: Bearer ' . base64_encode( $this->tokenDetails->token );
+        }
+
+        // RSA7e2
+        if ( ! empty( $this->defaultAuthOptions->clientId ) ) {
+            $headers[] = 'X-Ably-ClientId: ' . base64_encode( $this->defaultAuthOptions->clientId );
```

Comment:
Will this line overwrite the value of $`headers` and prevent the Authorization headers from being sent?

Reply:
This is the PHP syntax for appending to an array:

```
$ php -a
Interactive shell

php > $headers = [];
php > $headers[] = 'Authorization: Bearer ' . 'TOKEN';
php > $headers[] = 'X-Ably-ClientId: ' . 'clientId';
php > print_r ($headers);
Array
(
    [0] => Authorization: Bearer TOKEN
    [1] => X-Ably-ClientId: clientId
)
```

Reply:
oh nice, I stand corrected